app/src/main/java/org/oppia/android/app/home/HomeViewModel.kt

```diff
@@ -129,15 +157,28 @@ class HomeViewModel(
    * ongoing stories at all.
    */
   private fun computePromotedStoryViewModelList(
-    ongoingStoryList: OngoingStoryList
+    recommendedStoryList: RecommendedStoryList
   ): List<PromotedStoryViewModel> {
-    val storyList = if (ongoingStoryList.recentStoryCount != 0) {
-      ongoingStoryList.recentStoryList
-    } else {
-      // TODO(#936): Optimise this as part of recommended stories.
-      ongoingStoryList.olderStoryList
+    val storyList = when {
+      recommendedStoryList.suggestedStoryCount != 0 -> {
```

Comment:
Can we use the `let` here as well as right now all i could read in this file was 15 times `recommendedStoryList` :D

Reply:
:) Done.